### DISCUSSION TITLE
Could you please share with more examples that use MACE in LAMMPS?

### DISCUSSION THREAD

**Discussion by waywaygirl** [ACEsuit/mace#723] [Show and tell]
For example, the in file and the model data that you summarized in the paper.

I believe it will help more people to use MACE.

**Comment by gabor1** [comment]
please upload your error/logs/inputs here so that the people who help you have access to them.

**Comment by gabor1** [comment]
and your output and error...

**Comment by waywaygirl** [comment]
[in.simplify.txt](https://github.com/user-attachments/files/16053985/in.simplify.txt)
[water.data.txt](https://github.com/user-attachments/files/16053986/water.data.txt)
[log-55883.out.txt](https://github.com/user-attachments/files/16053988/log-55883.out.txt)
[log-55883.error.txt](https://github.com/user-attachments/files/16053989/log-55883.error.txt)

**Comment by waywaygirl** [comment]
Could any one help me to debug?

**Comment by wcwitt** [comment]
Please send the log/output you see for a single MPI process (e.g., launched with `mpirun -np 1`).

Also, do you have access to GPUs, or only CPUs?
